test/client/fixtures/sandbox/event/data-transfer-test.js

```diff
@@ -0,0 +1,253 @@
+var isIE11        = hammerhead.utils.browser.isIE11;
+var DataTransfer  = hammerhead.eventSandbox.DataTransfer;
+var DragDataStore = hammerhead.eventSandbox.DragDataStore;
+
+test('DataTransfer interface', function () {
```

Comment:
`DataTransfer` is a class, so it would be wrong to use lower case for it. But I'll update other test names